src/sentry/issues/escalating/forecasts.py

```diff
@@ -7,13 +7,13 @@
 from datetime import datetime
 
 from sentry import analytics
-from sentry.issues.escalating import (
+from sentry.issues.escalating.escalating import (
```

Comment:
here and elsewhere, `sentry.issues.escalating.escalating_xxx` feels a bit redundant - for the other paths, can we drop the `escalating_` prefix? and the logic here is mostly processing, so either helpers or processor might be a better name? 

okay to do as a followup if that's easier

Reply:
will do as a follow up!